main.c

```diff
@@ -702,7 +705,8 @@ void syncRecords() {
   );
   #endif
 }
-
+*/
```

Comment:
This doesn't have to be disabled: It only does stuff on `__EMSCRIPTEN__` anyway.